File: specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2024-03-01/examples-java/amlFilesystems_Delete.java

/**
 * Samples for AmlFilesystems Delete.
 */
public final class Main {
    /*
     * x-ms-original-file:
     * specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2024-03-01/examples/
     * amlFilesystems_Delete.json
     */
    /**
     * Sample code: amlFilesystems_Delete.
     * 
     * @param manager Entry point to StorageCacheManager.
     */
    public static void amlFilesystemsDelete(com.azure.resourcemanager.storagecache.StorageCacheManager manager) {
        manager.amlFilesystems().delete("scgroup", "fs1", com.azure.core.util.Context.NONE);
    }
}

File: specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2023-05-01/examples-js/amlFilesystems_Delete.js
const { StorageCacheManagementClient } = require("@azure/arm-storagecache");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Schedules an AML file system for deletion.
 *
 * @summary Schedules an AML file system for deletion.
 * x-ms-original-file: specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2023-05-01/examples/amlFilesystems_Delete.json
 */
async function amlFilesystemsDelete() {
  const subscriptionId =
    process.env["STORAGECACHE_SUBSCRIPTION_ID"] || "00000000-0000-0000-0000-000000000000";
  const resourceGroupName = process.env["STORAGECACHE_RESOURCE_GROUP"] || "scgroup";
  const amlFilesystemName = "fs1";
  const credential = new DefaultAzureCredential();
  const client = new StorageCacheManagementClient(credential, subscriptionId);
  const result = await client.amlFilesystems.beginDeleteAndWait(
    resourceGroupName,
    amlFilesystemName
  );
  console.log(result);
}
